man/repart.d.xml

```diff
@@ -787,6 +787,36 @@
 
         <xi:include href="version-info.xml" xpointer="v256"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>PreferESP=</varname></term>
```

Comment:
i wonder if we can make this more generic, i.e. FallbackFor= and then refer to some other partition. This would then mean that this partition is only created if the referenced partition cannot be created with the parameters defined, and otherwise suppressed.

Or something like that?

Reply:
or "UnlessExists="?